plugins/module_utils/crypto/_obj2txt.py

```diff
@@ -10,8 +10,7 @@
 #
 # The Apache 2.0 license has been included as LICENSES/Apache-2.0.txt in this collection.
 # The BSD License license has been included as LICENSES/BSD-3-Clause.txt in this collection.
-# SPDX-License-Identifier: Apache-2.0
-# SPDX-License-Identifier: BSD-3-Clause
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

Comment:
This looks good and matches upstream's license text:

> This software is made available under the terms of *either* of the licenses
found in LICENSE.APACHE or LICENSE.BSD. Contributions to cryptography are made
under the terms of *both* these licenses.